### PR TITLE
added 6.3.8

### PR DESCRIPTION
**Pull request by boinary**
check this code and merge to master

### CODE REVIEW

File: CodeHS/6/3/8/Area-of-a-square-with-default-parameters-Travis.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def calculate_area(side_length = 10):
+    print side_length
+    print side_length * side_length
+
+user_length = int(input("Enter side length: "))
+
+if user_length <= 0:
+    calculate_area()
+else:
+    calculate_area(user_length)
```
Comment (KenwoodFox) [inline CodeHS/6/3/8/Area-of-a-square-with-default-parameters-Travis.py]: You may want to add a propper EOF here but this otherwise looks good

**Review by KenwoodFox — APPROVED** [review]
Code looks good, may want to add EOF

File: CodeHS/6/3/8/Area-of-a-square-with-default-parameters-Travis.py (hunk 1 above)
Comment (boinary) [reply CodeHS/6/3/8/Area-of-a-square-with-default-parameters-Travis.py]: What would be an EOF in this? 

also the codehs compiler is having a hard time with a few things even though my code is logically correct
image

Comment (KenwoodFox) [reply CodeHS/6/3/8/Area-of-a-square-with-default-parameters-Travis.py]: CodeHS is dumb, and takes weird fiddling to make it work, your code is fine so id say dont stress yourself, just fiddle with extra text till it functions, codeHS does not expect elegant solutions
just press enter at the bottom of your file so you have a blank line

**Pull request merged**